eHealthAfrica/aether
# Aet 174 ui routing

#### What does this PR do?
Proxies ui requests to the kernel through kong using the configured kong auth-key
Setup `react-redux` store on the ui module
Setup `ui-routing` using `react-router-dom` 
Re configures kongfig for external access  

#### How to test
Run ui module `docker-compose up --build ui`
Check that kernel can be reached on `ui.aether.local/v1/kernel/`
Check that the reactjs app loads on `ui.aether.local:8004`
Check that you can navigate to `piplines` sample page using `/piplines`

#### Related JIRA ticket(s)
[AET-174](https://jira.ehealthafrica.org/browse/AET-174)
[AET-153](https://jira.ehealthafrica.org/browse/AET-153)
